easybuild/easyconfigs/f/foss/foss-2017b.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'Toolchain'
+
+name = 'foss'
+version = '2017b'
+
+homepage = '(none)'
+description = """GNU Compiler Collection (GCC) based compiler toolchain, including
+ OpenMPI for MPI support, OpenBLAS (BLAS and LAPACK support), FFTW and ScaLAPACK."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+gccver = '6.3.0-2.28'
```

Comment:
GCC 6.4.0 is imminent: https://gcc.gnu.org/ml/gcc/2017-06/msg00101.html (hat tip @ocaisa)